<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="verticalButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Up/Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="horizontalButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Left/Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>83</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="verticalGroupBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>120</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Vertical</string>
    </property>
    <widget class="QRadioButton" name="upRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>105</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="downRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>105</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="horizontalGroupBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>120</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Horizontal</string>
    </property>
    <widget class="QRadioButton" name="leftRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>105</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rightRadioButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>105</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>401</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select direction:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Another way to select direction:</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
